bigframes/functions/remote_function.py

```diff
@@ -198,9 +201,26 @@ def func(*bigframes_args, **bigframes_kwargs):
         )
     func.input_dtypes = tuple(function_input_dtypes)  # type: ignore
 
+    # set output bigframes data types
+    ibis_output_type = ibis_signature.output_type
+    if output_type is not None:
+        if not isinstance(ibis_signature.output_type, ibis.expr.datatypes.String):
+            raise TypeError(
```

Comment:
@tswast mentioned offline: "I would find that to be pretty annoying as a user. Better to make sure it matches and maybe warn that it's not necessary"

Reply:
Removed output_type from read_gbq_function, PTAL